proto/config.proto

```diff
@@ -681,6 +687,13 @@ message InputHistoryOptions
 	optional uint32 row = 4;
 }
 
+message GifOptions
+{
+	optional bool enabled = 1;
+	optional uint32 startingGifAddress = 2;
```

Comment:
yeah again modifying the address of EEPROM inside of an add-on will definitely break things.

Reply:
The address of the EEPROM is not being modified with this version of the code this is the flash memory layout

Code | GIF_CODE_DISTANCE| Empty space | Gif stuff | GIF_EEPROM_DISTANCE | EEPROM |